dkron-processor-files crash

I  dowaload dkron_0.9.3 (Mac 64bit and linux 64bit)  from https://github.com/victorcoder/dkron/releases

As long as I use dkron-processor-files to output log,  dkron crash.

Here is the error log of Mac version (my local machine):
```
INFO[2017-04-06T12:16:30+08:00] agent: Starting job                           job=test2 node=Dantes-MacBook-Pro.local
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x157a7c3]

goroutine 1316 [running]:
github.com/victorcoder/dkron/dkron.(*RPCServer).ExecutionDone(0xc42000e388, 0xc42044f6e0, 0x5, 0xed077b81e, 0xc96650e, 0x1a72f00, 0xed077b81e, 0xd01a178, 0x1a72f00, 0x1, ...)
        /Users/victorcoder/src/github.com/victorcoder/dkron/dkron/rpc.go:70 +0x7e3
reflect.Value.call(0xc420327200, 0xc42000e3a0, 0x13, 0x171d65d, 0x4, 0xc420305f20, 0x3, 0x3, 0xc420330e00, 0x0, ...)
        /usr/local/opt/go/libexec/src/reflect/value.go:434 +0x91f
reflect.Value.Call(0xc420327200, 0xc42000e3a0, 0x13, 0xc420022720, 0x3, 0x3, 0x1050960, 0xc420412420, 0xc420022758)
        /usr/local/opt/go/libexec/src/reflect/value.go:302 +0xa4
net/rpc.(*service).call(0xc42032cd80, 0xc420110080, 0xc420457450, 0xc42031bb00, 0xc420513e60, 0x16ba560, 0xc420600e00, 0x199, 0x15d6b60, 0xc42039fec0, ...)
        /usr/local/opt/go/libexec/src/net/rpc/server.go:387 +0x144
created by net/rpc.(*Server).ServeCodec
        /usr/local/opt/go/libexec/src/net/rpc/server.go:481 +0x404
```

And I create my job like this, 
```
data = {
    'name': 'test2',
    'schedule': '@every 10m',
    'command': '/bin/ls',
    'tags': {
        'role': 'eru:1'
    },
    'owner': '',
    'processors': {
        'file':{
            'forward': True,
            'log_dir': '/Users/ricebook/temp/'
        }
    }
}
```

--- a/dkron/rpc.go
+++ b/dkron/rpc.go
@@ -66,9 +66,10 @@ func (rpcs *RPCServer) ExecutionDone(execution Execution, reply *serf.NodeRespon
 	origExec := execution
 	for k, v := range job.Processors {
 		log.WithField("plugin", k).Debug("rpc: Processing execution with plugin")
-		processor := rpcs.agent.ProcessorPlugins[k]
-		e := processor.Process(&ExecutionProcessorArgs{Execution: origExec, Config: v})
-		execution = e
+		if processor, ok := rpcs.agent.ProcessorPlugins[k]; ok {
+			e := processor.Process(&ExecutionProcessorArgs{Execution: origExec, Config: v})
+			execution = e
+		}
 	}
 
 	// Save the execution to store

